### PR TITLE
Feedpage functionality 

### PR DESCRIPTION
**Pull request by vishal804**
Hello 

Created FeedPage Functionality 

### Functionality Created in this

Create Post 
Edit post 
Delete Post
Like Dislike post
Bookmark post 

### Files to review

src/component/postDisplay/PostDisplay.js
src/component/postEditModal/PostEditModal.js
src/pages/bookmarkPage/BookmarkPage.js
src/pages/feedPage/FeedPage.js
src/pages/profilePage/ProfilePage.js
src/redux/reducer/postsSlice.js

### CODE REVIEW

File: src/component/postDisplay/PostDisplay.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          </div>
+        </div>
+        <div className="post-content">
+          <p>{content}</p>
+        </div>
+        <div className="post-icons">
+          {isLike ? (
+            <button onClick={() => dispatch(dislikePost(_id))}>
+              <i className="fas fa-heart fa-lg"></i>
+              <span style={{ fontSize: "1rem" }}> {likeCount}</span>
```
Comment (Developers-codz) [inline src/component/postDisplay/PostDisplay.js:50]: try to avoid inline css

File: src/component/postEditModal/PostEditModal.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import "./postEditModal.css";
+import { useDispatch } from "react-redux";
+import { editPost } from "../../redux/reducer/postsSlice";
```
Comment (Developers-codz) [inline src/component/postEditModal/PostEditModal.js:4]: Suggestion - can use absolute path

**Review by Developers-codz — APPROVED** [review]
Code is readable and well structure, Good work on UI as well
Added a few suggestions, else good to go !!!

File: src/component/postEditModal/PostEditModal.js (hunk 2 above)
Comment (vishal804) [reply src/component/postEditModal/PostEditModal.js:4]: will try that

File: src/component/postDisplay/PostDisplay.js (hunk 1 above)
Comment (vishal804) [reply src/component/postDisplay/PostDisplay.js:50]: will do

**Comment by vishal804** [comment]
> Code is readable and well structure, Good work on UI as well Added a few suggestions, else good to go !!!

Thank You For Review

**Pull request merged**